src/mult.jl

```diff
@@ -29,6 +29,19 @@ include("cmult.jl")
 include("ncmult.jl")
 
 MP.multconstant(α, x::Monomial)   = MP.term(α, MA.mutable_copy(x))
+
+function zero_with_variables( ::Type{Polynomial{C,T}}, vars :: Vector{PolyVar{C}} ) where{C, T}
+    Polynomial( T[], emptymonovec(vars) )
```

Comment:
Remove spaces before and after parenthesis to stay consistent with the rest of the code base